static/spec/OpenLineage.yml

```diff
@@ -1,7 +1,7 @@
 openapi: 3.0.2
 info:
   title: OpenLineage
-  version: 0.0.1
+  version: 2-0-2
```

Comment:
Is there a particular reason why this value is used?

Reply:
this shouldn't be within the PR - no particular reason.